### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by awrelll** [tsega/meteor-mentions#1]
Is this package up and running? Can i use it?

**Comment by tsega** [comment]
Yes, it's working. Forgot to add the proper README, I'll do that pronto!

**Comment by tsega** [comment]
@awrelll please see the README now. You need to do some work on how you provide the list of user names. Any ideas on improving the package are welcome!
